sdks/java/extensions/avro/src/main/java/org/apache/beam/sdk/extensions/avro/schemas/utils/AvroUtils.java

```diff
@@ -630,7 +630,7 @@ public static SimpleFunction<Row, byte[]> getRowToAvroBytesFunction(Schema beamS
   }
 
   private static class RowToAvroBytesFn extends SimpleFunction<Row, byte[]> {
-    private final transient org.apache.avro.Schema avroSchema;
+    private final org.apache.avro.Schema avroSchema;
```

Comment:
also this change

Reply:
if the field is left transient there's a spotbugs warning that it's not being properly restored on deserialization - from what I see the Schema class became Serializable in the updated version of the avro, so we should be able to just serialize it and there's no need to mark it as transient anymore